packages/manager/cypress/support/api/vlans.ts

```diff
@@ -0,0 +1,27 @@
+import { VLAN, getVlans } from '@linode/api-v4';
+import { pageSize } from 'support/constants/api';
+import { depaginate } from 'support/util/paginate';
+
+import { isTestLabel } from './common';
+
+/**
+ * Returns a VLAN to use for a test resource, creating it if one does not already exist.
+ *
+ * @returns Promise that resolves to existing or new VLAN.
+ */
+export const findOrCreateDependencyVlan = async (linodeRegion: string) => {
+  const vlans = await depaginate<VLAN>((page: number) =>
+    getVlans({ page, page_size: pageSize })
+  );
+
+  const suitableVlans = vlans.filter(({ label, region }: VLAN) => {
+    return isTestLabel(label) && region === linodeRegion;
+  });
+
+  if (suitableVlans.length > 0) {
+    return suitableVlans[0];
+  }
```

Comment:
```suggestion
  const suitableVlan = vlans.find(({ label, region }: VLAN) => {
    return isTestLabel(label) && region === linodeRegion;
  });

  if (suitableVlan) {
    return suitableVlan;
  }
```
Very small suggestion: Using `Array.find` here might be a little easier to read later, but this functions exactly the same way as what you already have!